src/buck_vs_ldo.tex

```diff
@@ -121,7 +123,7 @@ \subsubsection{Operation}
     \label{fig:buck-dcm}
 \end{figure}
 
-\noindent When $I_L \approx 0$ the switch node voltage, $V_{sw}$ begins to oscillate with a peak of $\approx V_{out} \cdot 2$ converging towards $V_{out}$.\footnote{If the oscillation was undamped it would have a peak of exactly $V_{out} \cdot 2$.} This is because when $I_L \approx 0$, $V_{sw} = 0$, however, the output node is not at zero, $V_{out} \neq 0$ which means current will begin to flow through the inductor into the parasitic capacitance, $C_{parasitic}$ that exists between $V_{sw}$ and ground. This small resonant current, $I_L$, results in a noticable voltage fluctuation on $V_{sw}$, but not on $V_{out}$ because $C_{out} >> C_{parasitic}$ (where $C_{out}$ is the output capacitance of the buck converter). 
+\noindent When $I_L \approx 0$ the switch node voltage, $V_{sw}$ begins to oscillate with a peak of $\approx V_{out} \cdot 2$ converging towards $V_{out}$.\footnote{If the oscillation was undamped it would have a peak of exactly $V_{out} \cdot 2$ and would not decrease in amplitude over time. In this case damping is mostly provided by the parasitic resistance of the inductor.} This is because when $I_L \approx 0$, $V_{sw} = 0$, however, the output node is not at zero, $V_{out} \neq 0$ which means current will begin to flow through the inductor into the parasitic capacitance, $C_{parasitic}$ that exists between $V_{sw}$ and ground. This small resonant current, $I_L$, results in a noticable voltage fluctuation on $V_{sw}$, but not on $V_{out}$ because $C_{out} >> C_{parasitic}$ (where $C_{out}$ is the output capacitance of the buck converter). 
```

Comment:
comma after: If the oscillation was undamped
comma after: In this case

Reply:
This is because when $I_L \approx 0$, $V_{sw} = 0$, however, the output node is not at zero,

^ awkward however